lighthouse-core/test/report/html/renderer/report-ui-features-test.js

```diff
@@ -239,4 +239,19 @@ describe('ReportUIFeatures', () => {
       assert.ok(container.querySelector('.score100'), 'has fireworks treatment');
     });
   });
+
+  describe('metric descriptions', () => {
+    it('with no errors, hide by default', () => {
+      const lhr = JSON.parse(JSON.stringify(sampleResults));
+      const container = render(lhr);
+      assert.ok(!container.querySelector('.lh-metrics-toggle__input').checked);
```

Comment:
these assertions seem questionable..

Reply:
as in, I'm very explicitly testing against the if statement. idk seems odd

Reply:
yeah it feels like we really just want to assert that the errors are visible, do we do any gCS in these UI tests yet?

Reply:
gCS?

I first tried to do something like "are the description elements visible", but I don't think we can do that b/c jsdom doesn't try to surface stuff like that. as in, there is no rendering/css engine.

Reply:
I thought styles directly applied to elements still resolved though? I could be wrong I just thought there was no cascading.

Reply:
Ah, so I was actually thinking of _layout_ (from reading [this](https://github.com/jsdom/jsdom/issues/1048#issuecomment-76524044)). Tried a bit more, but still no dice:

![image](https://user-images.githubusercontent.com/4071474/58374685-31db7a80-7ef7-11e9-9c89-2ad14bbd44b4.png)

Reply:
LOL I forgot we shim getComputedStyles in this test. I thought it was provided by jsdom. anyways, idk how to do this.

Reply:
so if we don't overwrite their `getComputedStyles`, this happens:

![image](https://user-images.githubusercontent.com/4071474/58374844-38b7bc80-7efa-11e9-961d-32006ff83e1b.png)

weird